### PR TITLE
99 auth phone number

### PR DESCRIPTION
**Pull request by caudaganesh**
#99 

### Breakdown:
- [x] Add account kit for login and register with mobile phone
- [x] Modify redux to match account kit
- [x] Modify some components and styling to match account kit

### CODE REVIEW

**Comment by andy-shi88** [comment]
@caudaganesh there are conflicts, please fix them first.

**Comment by caudaganesh** [comment]
okay will do it tonight sir

File: app/containers/RegisterPhone/actions.js [hunk 1]
```diff
@@ -14,6 +15,11 @@ import {
   UPDATE_REGISTER_STATUS
 } from './constants';
 
+const ROLES = {
```
Comment (andy-shi88) [inline app/containers/RegisterPhone/actions.js]: please move it to `constants.js`

File: app/containers/RegisterPhone/actions.js [hunk 2]
```diff
@@ -79,23 +85,25 @@ export function updateRegisterStatus(status) {
...
+        first_name, last_name, username, email, password, role, social_id, provider, token
       }).then((response) => {
-        if (response && response.data && response.data.success) {
+        if (response && response.data && response.data.meta && response.data.meta.success) {
           // do something
           dispatch(updateRegisterStatus(true));
+          Alert.alert('Congratulation ' + first_name + ' Registration success (-_-)/ ');
         }
       }).catch((error) => {
+        Alert.alert('Registration failed (T_T)');
```
Comment (andy-shi88) [inline app/containers/RegisterPhone/actions.js]: should return some condition to `view` that trigger this alert in the `index` and remove the emoticon btw 😆 @caudaganesh

File: app/containers/RegisterPhone/index.js [hunk 3]
```diff
@@ -23,17 +30,55 @@ import * as actions from './actions';
...
 class RegisterPhone extends Component {
+
+  state = {
+    isEmailValid: false
+  };
   /*
      * initialize some state
      */
   componentWillMount() {
+    console.log(this.props)
```
Comment (andy-shi88) [inline app/containers/RegisterPhone/index.js]: clean the log

File: app/containers/RegisterPhone/index.js [hunk 4]
```diff
@@ -23,17 +30,55 @@ import * as actions from './actions';
...
+
     this.props.updateInputFields('role', 'attendee');
+    this.configureAccountKit();
+  }
+
+
+  onLogin(token) {
+    if (!token) {
+      console.warn('User canceled login')
```
Comment (andy-shi88) [inline app/containers/RegisterPhone/index.js]: clean the log

File: app/containers/RegisterPhone/index.js [hunk 5]
```diff
@@ -42,112 +87,168 @@ class RegisterPhone extends Component {
...
+        <Text style={styles.buttonText}>
+          FIELDS ARE NOT COMPLETE
+        </Text>
+      </Button>
+    );
+  }
+
+  render() {
+    if (this.props.isRegistering) {
+      console.log('isregistering...');
```
Comment (andy-shi88) [inline app/containers/RegisterPhone/index.js]: again the log.

**Review by andy-shi88 — CHANGES_REQUESTED** [review]
@caudaganesh  have a look

**Review by andy-shi88 — APPROVED** [review]

**Pull request merged**